src/dlgTriggerEditor.cpp

```diff
@@ -8407,6 +8409,7 @@ void dlgTriggerEditor::slot_clearSoundFile()
 {
     mpTriggersMainArea->lineEdit_soundFile->clear();
     mpTriggersMainArea->toolButton_clearSoundFile->setEnabled(false);
+    mpTriggersMainArea->lineEdit_soundFile->setToolTip(tr("<p>Sound file to play when the trigger fires.</p>"));
```

Comment:
Just to be clear - at this point there is NOT a sound file to play when the trigger fires because the user has just clicked the button to *clear* the option... :wink:

Reply:
Yep, and it's says "no file" then.